x-pack/solutions/observability/plugins/apm/test/scout/ui/fixtures/page_objects/transaction_details.ts

```diff
@@ -43,4 +99,51 @@ export class TransactionDetailsPage {
     await searchBar.fill(query);
     await searchBar.press('Enter');
   }
+
+  // Waterfall methods
+
+  /**
+   * Get all waterfall accordion items
+   */
+  async getWaterfallItems() {
```

Comment:
I don't see this being used 🤔 Did I miss it somewhere, or is the test not added yet? If we plan to add the test, it's Ok to have it; otherwise I would suggest removing it to avoid confusion that we use in the tests.